### ISSUE TITLE
Count up Timer can't set custom time to start with

### ISSUE THREAD

**Issue by RevealedSoulEven** [ashmanix/obs-plugin-countdown#104]
I am recording a timelapse
After recording 3 hours up the time I accidentally reset the timer, now I can't set it again to that time and start up, if even I set it to the desired counter by removing count up option and then set my time and seitch back to count up, its also not working,

**Comment by RevealedSoulEven** [comment]
I got a way to do it.

I saw that the text box is the upper time limit till when the timer should go that's why when I switched to count down and reset it and come back and hit play, it doen't change.

So what I did was to go to count down mode then reset it to desired time and then come back to count up mode and change the time in box to a high value and it starts counting up.

The only changes required it, it would be better if the reset button in countdown mode sets the timer to the one in the box and not to 00:00:00

**Comment by ashmanix** [comment]
Hi, thanks for the suggestion. I've kept the reset button for the timer for `period` times to be a hard reset for simplicity. It's why I had the pause button feature which in count up mode should pause the timer with the play button then starting the timer and keeping the current time thats on the display (in the box). Not sure if that helps at all?

**Comment by RevealedSoulEven** [comment]
The thing is if I pause the timer then close OBS, and come back again after
a restart, the resume button no more works. So I need to do the steps
mentioned to use that count up bug as a feature.

**Comment by ashmanix** [comment]
Yeah, when OBS is restarted the internal timers are reset to zero.

**Comment by RevealedSoulEven** [comment]
So fix wen :(

**Comment by ashmanix** [comment]
I do that by design. Are you wanting the current timer time to be stored and then loaded again on restart?

I'll need to double check the effects of that.

**Comment by RevealedSoulEven** [comment]
Yes kind of
Actually I need to pause the recording and between and again start from
there. Now the thing is it is showing the timer on next restart but not
continuing from there so it seems awkward.

**Comment by ashmanix** [comment]
I've made an update where the timer time is stored and loaded so you should hopefully be able to carry on from where you left off (for `period` type time only). You can check if it works in version 2.0.6:

https://github.com/ashmanix/obs-plugin-countdown/releases/tag/2.0.6

Tell me how it goes. Closing this issue for now but you can reopen the ticket if there any issues.

**Comment by RevealedSoulEven** [comment]
> Tell me how it goes. 

Thanks a lot sir. It's fine now!
